### PR TITLE
production file is renamed and production.rb is added in gitignore

### PR DESCRIPTION
**Pull request by namangupta01**
#1666

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,4 @@ pickle-email-*.html
 .env.local
 docker-compose.env
 docker-compose.yml
+production.rb
```
Comment (differentreality) [inline .gitignore]: Please add an empty line at the end of the file to make this red symbol disappear

Comment (namangupta01) [reply .gitignore]: okay!

**Comment by differentreality** [comment]
@hennevogel My thinking is that the users who use OSEM, change production.rb to whatever they need, then if they pull newest OSEM changes there might be conflicts.

But this really came up when I checked out https://github.com/SeaGL/osem/pull/6 to see their changes  (because I did not want to clone their whole repo, just to see a few changes) and it overwrote my production.rb

What's your thinking?

**Comment by lagartoflojo** [comment]
Pulling the latest version of OSEM should not overwrite `production.rb`. If there are conflicting changes, then those conflicts will be noticed by git, and it will complain. I have never seen a Rails project with an example `production.rb` file either. Plus, this would need documentation updates before it gets merged.

@hennevogel let's try to keep our comments courteous please :)

**Comment by hennevogel** [comment]
@differentreality users shouldn't change production.rb. The only place they should configure the app are the environment variables. What did it overwrite for you?

**Comment by differentreality** [comment]
It overwrote several things seagl has changed, such as:
https://github.com/SeaGL/osem/blob/master/config/environments/production.rb#L36
https://github.com/SeaGL/osem/blob/master/config/environments/production.rb#L70
https://github.com/SeaGL/osem/blob/master/config/environments/production.rb#L71
https://github.com/SeaGL/osem/blob/master/config/environments/production.rb#L82

Anyways, closing this

**Pull request closed without merging**